nav2_system_tests/src/updown/CMakeLists.txt

```diff
@@ -0,0 +1,21 @@
+ament_add_gtest_executable(test_updown
```

Comment:
test_updown doesn't seem to have any gtest in it. It should just be added with a standard `add_executable` call.

Reply:
Yes, true. Done. This is just a stress test that doesn't need to be run with the normal CI tests.